data/virt_autotest/host_unattended_installation_files/autoyast/dev_host_15.xml.ep

```diff
@@ -41,7 +41,7 @@
       </listentry>
       % }
     </add_on_products>
-  </add-on-->
+  </add-on>
```

Comment:
How about dev_host_12.xml.ep? Now 12sp5 enters ltss, and needs LTSS add-on if no registration.

Reply:
Done